app/provider/app.go

```diff
@@ -498,7 +504,7 @@ func New(
 	// NOTE: Any module instantiated in the module manager that is later modified
 	// must be passed by reference here.
 	app.MM = module.NewManager(
-		genutil.NewAppModule(
+		wrapped_genutil.NewAppModule(
```

Comment:
Genutil and staking need to be wrapped, because they send validator updates to CometBFT and we need to filter those.